### PR TITLE
video playback

### PR DESCRIPTION
**Pull request by argylelabcoat**
Videos play back, but audio isn't totally correct

### CODE REVIEW

**Comment by argylelabcoat** [comment]
I have been using this project for testing the A/V playback issue: https://github.com/argylelabcoat/Minimal-Godot-Video-Example

**Comment by akien-mga** [comment]
It seems to fail building:
```
modules/theora/register_types.cpp:31:38: fatal error: resource_importer_theora.h: No such file or directory
 #include "resource_importer_theora.h"
                                      ^
```

File: modules/opus/stub/register_types.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+#include "register_types.h"
+
+// Dummy module as libvorbis is needed by other modules (theora ...)
```
Comment (akien-mga) [inline modules/opus/stub/register_types.cpp]: Why are those needed? They don't do anything.

Comment (argylelabcoat) [reply modules/opus/stub/register_types.cpp]: Quick hack to keep some of the dependencies the video player needs still linking. Maybe there is a way to get the video player scons files to mark these as necessary, but I was trying to focus on making videos play again first.

**Comment by hi-ogawa** [comment]
Hi @argylelabcoat. Are you still working on this feature? and if not, do you mind if I take your commits here to make another PR for this video playback issue?

**Comment by akien-mga** [comment]
Superseded by #12014.
